Dockerfile

```diff
@@ -1,28 +1,23 @@
-FROM pytorch/pytorch:2.0.1-cuda11.7-cudnn8-devel
+FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-devel AS builder
 
-ENV DEBIAN_FRONTEND=noninteractive \
-    PATH="/opt/miniforge3/bin:$PATH" \
-    TORCH_CUDA_ARCH_LIST="8.0+PTX"
+ENV DEBIAN_FRONTEND=noninteractive
+ENV CUDA_HOME=/usr/local/cuda
+# 8.0+PTX means all architectures with compute capability >= 8.0 are supported.
+# This includes all consumer GPUs from the Ampere (RTX 30xx) and Ada Lovelace (RTX 40xx) families.
+# If you want to support other GPUs, consult https://developer.nvidia.com/cuda-gpus
+ENV TORCH_CUDA_ARCH_LIST="8.0+PTX" 
 
 RUN apt update && \
     apt install -y --no-install-recommends tzdata git unzip wget libglib2.0-0 libgl1-mesa-glx && \
-    wget -O Miniforge3.sh https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh && \
-    bash Miniforge3.sh -b -p /opt/miniforge3 && \
-    rm Miniforge3.sh && \
     apt clean && rm -rf /var/lib/apt/lists/*
     
 
 WORKDIR /workspace
```

Comment:
Line 15-16 could be replace by:
`RUN git clone --recursive https://github.com/google-research/Splat-SLAM.git /workspace/Splat-SLAM`